go/packages/dev-mode/version.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I'd make it as plain text version (and name it `VERSION` in such case), it seems overkill to maintain it as JSON object